python/cuml/dask/ensemble/randomforestregressor.py

```diff
@@ -0,0 +1,327 @@
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from dask.distributed import Client, wait, default_client, get_worker
+from dask import delayed
+from cuml.common.handle import Handle
+import dask.dataframe as dd
+import numba.cuda
+import random
+from tornado import gen
+from toolz import first
+from cuml.ensemble import RandomForestRegressor as cuRFR
+import math
+    
+
+def parse_host_port(address):
+    if '://' in address:
+        address = address.rsplit('://', 1)[1]
+    host, port = address.split(':')
+    port = int(port)
+    return host, port
+
+@gen.coroutine
+def _extract_ddf_partitions(ddf):
```

Comment:
Use `dask_df_utils` from above

Reply:
Per our discussion, leaving it as it is.